hknweb/init_permissions.py

```diff
@@ -0,0 +1,44 @@
+from hknweb.candidate.models import Announcement, OffChallenge, BitByteActivity
+from hknweb.events.models import Event, Rsvp
+from django.contrib.auth.models import User, Group, Permission
+from django.contrib.contenttypes.models import ContentType
+
+candidate = Group.objects.get_or_create(name='candidate')[0]
```

Comment:
Depending on what you consider 'readable', an alternate style may be to destructure the result into the group and success variables:

```suggestion
candidate, _created = Group.objects.get_or_create(name='candidate')
```

With this style the _created has an underscore to signify it is unused; it's occasionally beneficial when I'm not sure I remember what the `[0]` is for.